web/src/features/hfiCalculator/components/HFIPageSubHeader.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+import { Button, FormControl, Grid } from '@material-ui/core'
+import FireCentreDropdown from 'features/hfiCalculator/components/FireCentreDropdown'
+import { isUndefined } from 'lodash'
+import { FireCentre } from 'api/hfiCalcAPI'
+import AboutDataModal from 'features/hfiCalculator/components/AboutDataModal'
+import { HelpOutlineOutlined } from '@material-ui/icons'
+import DatePicker from 'components/DatePicker'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    minHeight: 50,
+    marginBottom: '1rem',
+    paddingBottom: '0.5rem',
+    paddingTop: '0.5rem',
+    paddingLeft: '1rem',
+    paddingRight: '1rem',
+    fontSize: '1.3rem',
+    background: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    alignContent: 'center'
+  },
+  positionStyler: {
+    position: 'absolute',
+    right: '20px'
+  },
+  helpIcon: {
+    fill: 'white'
+  },
+  aboutButtonText: {
+    color: 'white',
+    textDecoration: 'underline',
+    fontWeight: 'bold',
+    justifyContent: 'flex-end'
+  },
+  dateOfInterestPicker: {
+    marginLeft: '7px',
+    alignItems: 'center',
+    '& .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    },
+    '& .MuiOutlinedInput-input': {
+      color: 'white'
+    },
+    '& .MuiIconButton-root': {
+      color: 'white'
+    },
+    '& .MuiInputLabel-root': {
+      color: 'white'
+    },
+    '& .MuiOutlinedInput-root': {
+      borderColor: 'white'
+    },
+    '&:hover .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    },
+    '&:focus .MuiOutlinedInput-notchedOutline': {
+      borderColor: 'white'
+    }
+  }
+}))
+
+interface Props {
+  padding?: string
+  formControlClass: string
```

Comment:
`HFIPageSubHeader` can just import the `formControl` class like other components do so you can save passing another prop.